src/main/resources/sql/select2-1-select2-5.sql

```diff
@@ -0,0 +1,35 @@
+-- Find the names of buyers, whose age is from 16 to 35
+SELECT c.name
```

Comment:
May be we should also select c.age in order to control results?

Reply:
The task is asked to find the names of customers